### PR TITLE
centos and ODSbox removal in ods core

### LINKED ISSUE #1209: Removal of Centos and deprecation of ODSBox
**Is your feature request related to a problem?**
Centos7 eol will be on June 2024. up until now we have been offering both, ubi and centos but we will only continue to support ubi images from now on.

**Describe the solution you'd like**
Remove Centos images and deprecate ODSBox

**Additional context**
This is needed in order to continue with the update to nodejs18 in the quickstarters -> https://github.com/opendevstack/ods-quickstarters/pull/795

### PR DESCRIPTION
**Pull request by braisvq1996**
Closes #1209 for ods-core

### CODE REVIEW

**Comment by metmajer** [comment]
Hello @braisvq1996! Thanks for providing the PR, as discussed with me and @michaelsauter. Do you plan to make more changes as this is still in Draft mode?

**Comment by braisvq1996** [comment]
I will left this as a draft until we are sure that all centos references have been remoded.
So far I did an initial scan and will continue other day.

In ods-quickstarters I have also prepared another PR for the removal of centos

**Comment by braisvq1996** [comment]
Also, as far as i know, ODSbox code was under the ods-devenv folder and I have removed it entirely

File: jenkins/ocp-config/build/bc.yml [hunk 1]
```diff
@@ -33,10 +33,10 @@ parameters:
   required: true
   description: Base image for Jenkins agent base
 - name: JENKINS_MASTER_DOCKERFILE_PATH
-  value: Dockerfile.centos7
+  value: Dockerfile.rhel7
```
Comment (michaelsauter) [inline jenkins/ocp-config/build/bc.yml:36]: Why is this not `ubi8`?

Comment (braisvq1996) [reply jenkins/ocp-config/build/bc.yml]: We have a rhel dockerfile for jenkins master. But we might as well remove it also and maintain ubi only

**Review by michaelsauter — APPROVED** [review]

**Pull request merged**